lib/devcycle-ruby-server-sdk/localbucketing/config_manager.rb

```diff
@@ -38,16 +41,16 @@ def initialize(sdkKey, local_bucketing, wait_for_init)
     def initialize_config
       begin
         fetch_config
-        @config_poller.execute if @polling_enabled
+        start_polling(false)
       rescue => e
         @logger.error("DevCycle: Error Initializing Config: #{e.message}")
       ensure
         @local_bucketing.initialized = true
       end
     end
 
-    def fetch_config
-      return unless @polling_enabled
+    def fetch_config(min_last_modified: -1)
+      return unless @polling_enabled || @sse_active && @enable_sse
```

Comment:
whats the order of operations here? is the `&&` evaluated first? maybe add brackets?

Reply:
Yeah - the and condition is operated first - I'll tag that